src/vs/amdX.ts

```diff
@@ -70,10 +70,8 @@ class AMDModuleImporter {
 		(globalThis as any).define.amd = true;
 
 		if (this._isRenderer) {
-			// eslint-disable-next-line no-restricted-globals
 			this._amdPolicy = (globalThis as any)._VSCODE_WEB_PACKAGE_TTP ?? window.trustedTypes?.createPolicy('amdLoader', {
```

Comment:
@bpasero I removed these as I don't see any rules that would apply to them anymore. In the new eslint version, having a suppression comment that does nothing is an error

 Let me know if this is incorrect and we still should be getting lint errors here

Reply:
@mjbvz but the rule is in use and actually catches very ugly uses of globals such as `event` or `length`:

https://github.com/microsoft/vscode/blob/de557e7b66cc8a085ae80f6b3f1e4b496a1f6e6c/.eslintrc.json#L39-L50

Reply:
Or are you saying, the rule is still there but just does not apply to this one file `amdX.ts`? That can well be true.

Reply:
Yes sorry this was unclear. The rule still seems to work but no lint error is reported on this specific line. I just wanted to double check with you that that was ok. Maybe `window` or `globalThis` was in the list of restricted names at some point?

Reply:
I think its more likely that `amdX` is not covered by ESlint rules anymore or for a while now given its rather unusual location in `src/vs`.